cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -146,8 +177,16 @@ cm_ig_parcels.html>`__, but in summary these are the steps:
    <http://www.cloudera.com/content/cloudera/en/documentation/core/latest/topics/cm_ig_parcels.html#concept_vwq_421_yk_unique_1__section_ug1_c3y_bm_unique_1>`__
    the parcel.
 
-If the Cask parcel repository is inaccessible to your cluster, please see :ref:`these
-suggestions <faqs-cloudera-direct-parcel-access>`.
+**Notes:**
+
+- If the Cask parcel repository is inaccessible to your cluster, please see :ref:`these
+  suggestions <faqs-cloudera-direct-parcel-access>`.
+- The CDAP parcels are hosted at a repository determined by the CDAP version.
+  For instance, the CDAP |version| parcels are hosted at the URL:
+  
+  .. parsed-literal::
+  
+    \http://repository.cask.co/parcels/cdap/|version|\ /
```

Comment:
In the rendered page, ``version`` was replaced with ``3.3.1``, but it should be ``3.3`` here